===================================================================
RCS file: /cvs/gnome/gnome-vfs/programs/gnomevfs-df.c,v
retrieving revision 1.1
retrieving revision 1.1.2.1
diff -u -r1.1 -r1.1.2.1
--- gnomevfs-df.c	2006/03/30 09:13:13	1.1
+++ gnome-vfs/programs/gnomevfs-df.c	2006/05/14 10:25:38	1.1.2.1
@@ -79,6 +79,11 @@
 	int i;
 
 	type = gnome_vfs_volume_get_filesystem_type (vol);
+
+	if (type == NULL) {
+		return TRUE;
+	}
+
 	for (i = 0; i < G_N_ELEMENTS (ignored_fs); i++) {
 		if (strcmp (ignored_fs[i], type) == 0) {
 			g_free (type);
